packages/contracts-bedrock/test/L2/L2StandardBridgeInterop.t.sol

```diff
@@ -0,0 +1,361 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Target contract is imported by the `Bridge_Initializer`
+import { Bridge_Initializer } from "test/setup/Bridge_Initializer.sol";
+import { console2 } from "forge-std/console2.sol";
+
+// Target contract dependencies
+import {
+    L2StandardBridgeInterop,
+    InvalidDecimals,
+    InvalidLegacyAddress,
+    InvalidSuperchainAddress,
+    InvalidTokenPair,
+    IOptimismMintableERC20Factory,
+    MintableAndBurnable
+} from "src/L2/L2StandardBridgeInterop.sol";
+import { IERC20Metadata } from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import { IERC165 } from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import { IOptimismMintableERC20 } from "src/universal/IOptimismMintableERC20.sol";
+
+// TODO: Replace Predeploys.OPTIMISM_SUPERCHAIN_ERC20_FACTORY with optimismSuperchainERC20Factory
+import { Predeploys } from "src/libraries/Predeploys.sol";
+
+contract L2StandardBridgeInterop_Test is Bridge_Initializer {
+    /// @notice Emitted when a conversion is made.
+    event Converted(address indexed from, address indexed to, address indexed caller, uint256 amount);
+
+    /// @notice Mock ERC20 decimals
+    function _mockDecimals(address _token, uint8 _decimals) internal {
+        vm.mockCall(_token, abi.encodeWithSelector(IERC20Metadata.decimals.selector), abi.encode(_decimals));
+    }
+
+    /// @notice Mock ERC165 interface
+    function _mockInterface(address _token, bytes4 _interfaceId, bool _supported) internal {
+        vm.mockCall(
+            _token, abi.encodeWithSelector(IERC165.supportsInterface.selector, _interfaceId), abi.encode(_supported)
+        );
+    }
+
+    /// @notice Mock factory deployment
+    function _mockDeployments(address _factory, address _token, address _deployed) internal {
+        vm.mockCall(
+            _factory,
+            abi.encodeWithSelector(IOptimismMintableERC20Factory.deployments.selector, _token),
+            abi.encode(_deployed)
+        );
+    }
+}
+
+/// @notice Test when converting from a legacy token to a SuperchainERC20 token
```

Comment:
This line should go one behind, just above the test function

Reply:
This was specific for the test contract, but i will add a new one to the function below